File: 02-07-2021/sebaGato/app/src/main/java/com/example/sebagato/MainActivity.java
package com.example.sebagato;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.sebagato.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding b;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        b = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(b.getRoot());

        b.reiniciarBtn.setOnClickListener( v -> {

        });
    }
}

File: ContactosKotlin/app/src/main/java/com/example/contactoskotlin/MainActivity.kt
package com.example.contactoskotlin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.NavController
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.ui.NavigationUI
import com.example.contactoskotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    lateinit var binding : ActivityMainBinding
    lateinit var nav : NavController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val host = supportFragmentManager.findFragmentById(R.id.navFrag) as NavHostFragment?
        nav = host!!.navController

        NavigationUI.setupActionBarWithNavController(this, nav)

        with( binding ){

        }
    }
    private fun closeKeyboard(){

    }
}